src/ps/commands/joinphrases.ts

```diff
@@ -0,0 +1,103 @@
+import { addJoinphrase, deleteJoinphrase, getJoinphrase } from '@/database/joinphrases';
+import { ChatError } from '@/utils/chatError';
+
+import type { NoTranslate, ToTranslate } from '@/i18n/types';
+import type { PSCommand } from '@/types/chat';
+
+export const command: PSCommand = {
+	name: 'jp',
+	help: 'Joinphrases module!',
+	perms: ['room', 'driver'],
+	syntax: 'CMD',
+	aliases: ['joinphrase'],
+	categories: ['utility'],
+	extendedAliases: {
+		addjp: ['jp', 'new'],
+		deletejp: ['jp', 'delete'],
+		removejp: ['jp', 'delete'],
+		remjp: ['jp', 'delete'],
+		getjp: ['jp', 'get'],
+		showjp: ['jp', 'get'],
+		displayjp: ['jp', 'get'],
+	},
+	children: {
+		new: {
+			name: 'new',
+			help: 'Adds a new  joinphrase for a given user',
+			perms: ['room', 'driver'],
+			flags: { allowPMs: false },
+			syntax: 'CMD [User], [Joinphrase]',
+			aliases: ['add', 'a', 'n'],
+			async run({ message, arg, $T, checkPermissions }) {
+				if (!checkPermissions(['room', 'driver'])) {
+					throw new ChatError($T('ACCESS_DENIED'));
+				}
+				if (!arg) {
+					message.reply($T('INVALID_ARGUMENTS'));
+				}
+				const args: string[] = arg.split(',').map(s => s.trim());
+				const username = args[0];
+				const phrase = args[1];
```

Comment:
This is broken with commas (phrases with commas will only have the first bit)
```suggestion
				const [username, phrase] = arg.lazySplit(/\s*,\s*/, 1);
```

See https://github.com/PartMan7/PartBot/blob/main/src/globals/prototypes.ts#L58 for reference